kasa/smart/smartdevice.py

```diff
@@ -190,6 +198,7 @@ async def _initialize_features(self):
                     "Signal Level",
                     attribute_getter=lambda x: x._info["signal_level"],
                     icon="mdi:signal",
+                    category=Feature.Category.Debug,
```

Comment:
Maybe this should be `Info`

Reply:
Ah, yes, this would be doable. RSSI would be left to debug though, or hidden from homeassistant using a custom "extra metadata", as it's very noisy.

Reply:
Done.